docs/tutorials/tabular_prediction/tabular-indepth.md

```diff
@@ -61,14 +60,19 @@ hyperparameters = {  # hyperparameters of each model type
                    'NN': nn_options,  # NOTE: comment this line out if you get errors on Mac OSX
                   }  # When these keys are missing from hyperparameters dict, no models of that type are trained
 
-time_limits = 2*60  # train various models for ~2 min
+time_limit = 2*60  # train various models for ~2 min
 num_trials = 5  # try at most 3 different hyperparameter configurations for each type of model
 search_strategy = 'skopt'  # to tune hyperparameters using SKopt Bayesian optimization routine
 
-predictor = task.fit(train_data=train_data, tuning_data=val_data, label=label_column,
-                     time_limits=time_limits, eval_metric=metric, num_trials=num_trials,
-                     hyperparameter_tune=hp_tune, hyperparameters=hyperparameters,
-                     search_strategy=search_strategy)
+hyperparameter_tune_kwargs = {  # specify to enable HPO
```

Comment:
maybe clarify:
# HPO is not performed unless `hyperparameter_tune_kwargs` is specified

Reply:
Added